### DISCUSSION TITLE
Is there a way to add at ease paginations?

### DISCUSSION THREAD

**Discussion by GabrieleCalarota** [fastapi/fastapi#4663] [Questions]
I couldn't find any reference for pagination in FastAPI on the documentation. Is it a possible new feature like Django-Rest-Framework using HATEOAS?

**Comment by nikstuckenbrock** [comment]
Hey @GabrieleCalarota,

maybe the [fastapi-pagination](https://github.com/uriyyo/fastapi-pagination) library fits your needs?

**Comment by YuriiMotov** [comment]
There is no internal support for this.
You can either implement this by yourself or use external library
